x-pack/plugins/ml/server/models/data_recognizer/modules/security_linux/ml/v3_linux_anomalous_network_activity.json

```diff
@@ -0,0 +1,54 @@
+{
+  "job_type": "anomaly_detector",
+  "description": "Looks for unusual processes using the network which could indicate command-and-control, lateral movement, persistence, or data exfiltration activity.",
+  "groups": [
+    "auditbeat",
+    "endpoint",
+    "linux",
+    "network",
+    "security"
+  ],
+  "analysis_config": {
+    "bucket_span": "15m",
+    "detectors": [
+      {
+        "detector_description": "rare by \"process.name\"",
+        "function": "rare",
+        "by_field_name": "process.name"
+      }
+    ],
+    "influencers": [
+      "host.name",
+      "process.name",
+      "user.name",
+      "destination.ip"
+    ]
+  },
+  "allow_lazy_open": true,
+  "analysis_limits": {
+    "model_memory_limit": "64mb"
+  },
+  "data_description": {
+    "time_field": "@timestamp"
+  },
+  "custom_settings": {
+    "custom_urls": [
```

Comment:
Does it matter that these URLs use `siem#` rather than starting with `security`?

Reply:
@randomuserid any thoughts/preferences here?

Reply:
they still work in main. Only the UI labels were renamed from siem to security, not everything in the backend.